### PR TITLE
fixing infra test - do not merge until SDK release

### PR DESCRIPTION
**Pull request by roysagi**
Fixing infrastructure after the addition on `FileType.PACK_METADATA`.

Use this branch as SDK version release branch for content.

### CODE REVIEW

**Comment by Itay4** [comment]
@roysagi lets trigger nightly build as well

File: Tests/scripts/utils/get_modified_files_for_testing.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 class FileType(constants.FileType, Enum):
     CONF_JSON = "confjson"
-    METADATA = "metadata"
+    SECRETS_IGNORE = "secretsignore"
```
Comment (yaakovi) [inline Tests/scripts/utils/get_modified_files_for_testing.py:20]: why do we need it? this file should not affect any logics related to selected tests/packs

**Review by yaakovi — COMMENTED** [review]
should this be part of the release branch of the SDK?

**Comment by roysagi** [comment]
closing as this is covered by a different PR

**Pull request closed without merging**